x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/query/index.ts

```diff
@@ -356,10 +369,9 @@ export function registerQueryFunction({
           if (msg.message.function_call.name) {
             return msg;
           }
-          let esqlQuery = correctCommonEsqlMistakes(msg.message.content, resources.logger).match(
-            /```esql([\s\S]*?)```/
-          )?.[1];
-          esqlQuery = await correctQueryWithActions(esqlQuery ?? '');
```

Comment:
I have temporarily disabled this as I saw some incorrect fixes being applied which led to syntax errors. E.g.:

```
   | WHERE @timestamp >= NOW() - 15 minutes
```

became:
```
   | WHERE `@timestamp >= NOW(`) - 15 minutes
```